### DISCUSSION TITLE
(08) RistoBook [P.20220506]

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#46] [Progetti]
In questa discussion a tema il progetto [RistoBook](https://elearning.uniroma1.it/pluginfile.php/1350504/mod_folder/content/0/%285%29%20P%20-%20Progetti/%2808%29%20-%20Ristobook%20-%20BD2.pdf), discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**N.B.:** specifica dei requisiti del progetto in questione e del progetto omonimo, oggetto del testo di esame del 4 febbraio 2022, sono perfettamente identici

**Comment by progettistamatto** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/Exyss/university-notes/raw/main/Bachelor/Secondo%20Anno/Basi%20di%20Dati%20II/RistoBook.pdf) una soluzione proposta da Simone Bianco (Exyss)

> [!Warning]
_Le soluzioni di Simone Bianco fanno uso degli **Entity-Relationship (ER)** Diagrams su cui il corso si basava fino all'Anno Accademico 2023/2024, mentre **attualmente vengono utilizzati gli UML Class** Diagram. Sebbene i due siano estremamente simili, vi sono **alcune discordanze tra di loro**, in particolare l'**inversione di posizione dei vincoli di molteplicità agli estremi delle relationship**. Nonostante ciò, il rimanente delle fasi di analisi e progettazione delle soluzioni proposte risulta inaffetto da tale cambiamento. Per tanto, una volta comprese le differenze tra ER e Classi UML, le soluzioni risultano normalmente fruibili._

Rispondi a questo commento taggando @Exyss per eventuali dubbi o segnalazioni in merito

**Reply by Loriv3** [reply]
Stavamo lavorando su ristobook e per vedere confontarci abbiamo visto qui. Ci chiedevamo, cosa sono quei simboli in blu affianco ad alcuni vincoli?

**Reply by emanueleBu** [reply]
purtoppo la soluzione non è più disponibile

**Reply by progettistamatto** [reply]
> purtoppo la soluzione non è più disponibile

Risolto @emanueleBu, grazie per la segnalazione!

**Comment by progettistamatto** [comment]
Per questo progetto è disponibile [[QUI]](https://raw.githubusercontent.com/SimoneLid/BD2/main/Progetti/RistoBook/RistoBook.pdf) una soluzione proposta da Simone Lidonnici (@SimoneLid)

**Comment by daromacarlo** [comment]
https://github.com/daromacarlo/Triennale-in-Informatica/tree/main/Basi%20di%20dati/Basi%20di%20dati%20MODULO%202/Progetti/8.RistoBook

**Comment by CosaFaSelect1** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/CasuFrost/university-notes-Informatica/raw/main/Secondo%20Anno/Secondo%20Semestre/Basi%20di%20Dati%202/Progetti/RistoBook/RistoBook.pdf) una soluzione proposta da Marco Casu
